openedx/features/classrooms/models.py

```diff
@@ -0,0 +1,12 @@
+from nodebb.models import DiscussionCommunity
+from django.contrib.auth.models import User
+from django.db import models
+from model_utils.models import TimeStampedModel
```

Comment:
Kindly fix the import structure. And also consider using `isort` on your files.

Reply:
Done.